src/js/lib.js

```diff
@@ -726,16 +709,15 @@ vjs.parseUrl = function(url) {
  * @private
  */
 function _logType(type, args){
-  var argsArray, noop, console;
-
   // convert args to an array to get array functions
-  argsArray = Array.prototype.slice.call(args);
+  let argsArray = Array.prototype.slice.call(args);
   // if there's no console then don't try to output messages
   // they will still be stored in vjs.log.history
   // Was setting these once outside of this function, but containing them
   // in the function makes it easier to test cases where console doesn't exist
-  noop = function(){};
-  console = window['console'] || {
+  let noop = function(){};
```

Comment:
Didn't mean to shut down this thread with my comment if that happened. I am interested to know if there's benefits. I think the issue is that `noop` is a pretty ambiguous var name itself, so `noop = Function.prototype`, doesn't give you any clues unless you know one or the other.